internal/model/models.go

```diff
@@ -1,15 +1,13 @@
 package model
 
 import (
-	"time"
-
 	"github.com/G-Research/yunikorn-core/pkg/webservice/dao"
 )
 
-type ApplicationDAOInfo struct {
-	CreatedAt              time.Time `json:"createdAt"`
-	QueueID                string    `json:"queueId"`
-	dao.ApplicationDAOInfo `json:",inline"`
+type ModelMetadata struct {
```

Comment:
Why not name it just `Metadata`? It is clear it is a model from the package name

Reply:
I forgot to do that... Good catch!